### PR TITLE
Adding owners for `e2e` directory.

### PR DESCRIPTION
**Pull request by rhopp**
Signed-off-by: Radim Hopp <rhopp@redhat.com>

### What does this PR do?
Adds owners for `e2e` folder containing Hapy Path E2E tests.

### CODE REVIEW

**Review by dmytro-ndp — APPROVED** [review]
@Ohrimenko1988 had asked to remove him from `selenium/**` as well.

**Comment by Ohrimenko1988** [comment]
In my opinion, we should remove @dmytro-ndp from code owners of this project, because he doesn't involve to this project.

**Comment by dmytro-ndp** [comment]
@Ohrimenko1988 is right.
Fill free to remove me from the list of code owners of `e2e` project, @rhopp.

**Comment by rhopp** [comment]
@Ohrimenko1988 I've removed Dmytro. Is there anything else we agreed on, or is this ready to merge (after the checks of course :-D)?

File: .github/CODEOWNERS
Comment (benoitf) [inline .github/CODEOWNERS]: FYI there is a missing end of line

**Review by Ohrimenko1988 — APPROVED** [review]

**Pull request merged**
